src/libraries/Fuzzing/DotnetFuzzing/Dictionaries/typename.dict

```diff
@@ -0,0 +1,26 @@
+"	"
+"0"
+","
+"[*["
+" System.Int32  "
+"System.Int32&&"
+"System.Int32[,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,]"
+"System.Int32[*******************************]"
+"System.Int32[][][][][][][][][][][][][][][][][][][][]"
+".NoNamespace"
+"Namespace.Ko\x15B\x107" 
```

Comment:
```suggestion
"Namespace.Ko\xC5\x9B\xC4\x87" 
```
if you wanted "Namespace.Kość"

Reply:
Yes, I wanted "Namespace.Kość", I found those numbers from https://unicodelookup.com/#Kość/1